datasets/glossis/vtk_environment.yml

```diff
@@ -0,0 +1,19 @@
+name: env
+channels:
+  - conda-forge
+dependencies:
+  - netCDF4
+  - numpy
+  - scipy
+  - pandas
+  - matplotlib
+  - mayavi==4.7.2
+  - xarray
+  - gridded
+  - cmcrameri 
+  - rasterio
+  - earthengine-api
```

Comment:
Might still be good to pin all the versions, or at least the critical ones to a current working minor version.